src/v3/src/transformer/layout/idxTransformerMapping.ts

```diff
@@ -239,6 +247,13 @@ const TransformerMap: {
     },
   },
   [IDX_STEP.ENROLL_AUTHENTICATOR]: {
+    [AUTHENTICATOR_KEY.SYMANTEC_VIP]: {
+      transform: transformSymantecVipAuthenticator,
+      buttonConfig: {
+        showDefaultSubmit: false,
+        showReturnToAuthListLink: true,
```

Comment:
If I remember correctly, this is true by default, so you can remove this and it should work as expected.

Reply:
done